setup.py

```diff
@@ -2,12 +2,13 @@
 
 setup(
     name="edx-opaque-keys",
-    version="0.2.0",
+    version="0.2.1",
     author="edX",
     url="https://github.com/edx/opaque-keys",
     packages=[
         "opaque_keys",
         "opaque_keys.edx",
+        "opaque_keys.edx.tests",
```

Comment:
Perhaps it would be safest now to include everything.  "opaque_keys.tests" is still missing.